dev-tools/omdb/src/bin/omdb/main.rs

```diff
@@ -58,7 +61,38 @@ mod oxql;
 mod reconfigurator;
 mod sled_agent;
 
-const OMDB_UUID: Uuid = Uuid::from_u128(0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAu128);
+struct ConfirmationPrompt(Reedline);
```

Comment:
nit: I'd probably put this in helpers.rs.

Reply:
done in 8c3c39ba0